portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path

```diff
@@ -37,7 +37,7 @@
 </tr>
 <tr>
 	<td>CONTENT_CHOOSE_CONTENT_CONTENT_CHECKBOX</td>
```

Comment:
This path is used by other macros. Did you check if these changes won't change the behavior for other tests?